dnsmasq.conf

```diff
@@ -1,7 +1,7 @@
-interface=INTERFACE
+interface=PROVISIONING_INTERFACE
 bind-dynamic
 dhcp-range=DHCP_RANGE
-enable-tftp
+enable-tftp=PROVISIONING_INTERFACE
```

Comment:
Not sure we need this (man page - "Without an argument, the TFTP service is provided to the same set of interfaces as DHCP service"), but it doesn't hurt.

Reply:
yeah this was a leftover.